packages/now-next/test/fixtures/14-next-offline/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "next": "canary",
+    "next": "latest",
```

Comment:
Can we revert this change? We fixed this test with `next@canary`!

Reply:
Done!